src/toolchain/export-android.md

```diff
@@ -0,0 +1,105 @@
+<!--
+  ~ Copyright (c) godot-rust; Bromeon and contributors.
+  ~ This Source Code Form is subject to the terms of the Mozilla Public
+  ~ License, v. 2.0. If a copy of the MPL was not distributed with this
+  ~ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+-->
+
+NOTE: Keep track of where you install the Android Command Line Tools and JDK 17
+
+Exporting with gdext for godot requires some of the same pieces that are required for building godot from source.
+Specifically, the Android SDK Command Line Tools and JDK 17 as mentioned in Godot's documentation here: 
+https://docs.godotengine.org/en/stable/contributing/development/compiling/compiling_for_android.html#requirements
+
+Once you have those installed, you then need to follow godot's instructions for setting up the build system here:
+https://docs.godotengine.org/en/stable/contributing/development/compiling/compiling_for_android.html#setting-up-the-buildsystem
+
+As of writing this, I'm not entirely sure if JDK 17 is required in order for Android Command Line Tools to function properly.
+I have not tested without it.  To be safe, it is included in the documentation for now.
+
+Now on to the fun part of actually compiling gdext for Android! :)
+
+Set the environment variable CLANG_PATH to point to Android's build of clang.  Example:
```

Comment:
Use single backticks for symbols like `CLANG_PATH`.

Avoid double spaces (before "Example") -- you have several occurrences of it in other places, too.